tests/db.rs

```diff
@@ -0,0 +1,22 @@
+extern crate postgres;
+
+use self::postgres::{Connection, TlsMode};
+
+/// Wrapper that returns a PostgreSQL connection to the tests environment db
+pub fn get_connection() -> Connection {
+
+    return Connection::connect(
+        "postgresql://vagrant:vagrant@db:5432",
```

Comment:
it will not pass travis as it use different connection parameter, should get it from the environment too I think ?

Reply:
True. From the environment is find to me.